fetch.bs

```diff
@@ -2157,6 +2157,11 @@ Unless stated otherwise, it is false.
 
 <p class=note>This is for exclusive use by HTML's navigate algorithm. [[!HTML]]
 
+<p>A <a for=/>request</a> has an associated <dfn export for=request>navigation id</dfn>
+(null or a string). Unless stated otherwise, it is null.
+
+<p class=note>This is for exclusive use by HTML's navigate algorithm. [[!HTML]]
```

Comment:
What do you think about calling these fields "WebDriver navigation id" and "WebDriver id"? If we ever make them more generic we can rename them and the way WebDriver consumes them can just be as ID, but at least that would discourage others from using them for purposes they're not intended for without additional review.

Reply:
I don't really have a strong opinion so I changed them.